exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -334,6 +341,36 @@ public StoredConfiguration getActualConfiguration() {
     });
   }
 
+  @Override
+  public void initializeTimeSchema(View view) {
+    this.timeSchema = TimeSchema.newInstance(view);
```

Comment:
This code is broken — why TimeSchema is created by (whom? there are no prod usages) for a particular View (= an immutable Snapshot)? 

It shall be treated the same way a schema of any service is used in read requests:

```java
return node.withSnapshot(s -> {
  var timeOracle = TimeSchema.newInstance(s);
  var currentTime = timeOracle.getTime();
  return toOptional(currentTime);
});
```

As it turned out to be unclear — please update the documentation of TimeSchema.